boards/common/nrf52/bootloader_nrfutil.dep.mk

```diff
@@ -0,0 +1,6 @@
+include $(RIOTBOARD)/common/makefiles/stdio_cdc_acm.dep.mk
```

Comment:
Why not move this line inside the ifneq block below ?

Reply:
`stdio_cdc_acm` is added to `USEMODULE` by `stdio_cdc_acm.dep.mk` (if no other stdio has been selected by the user)

So if we move it, the ifneq would always be false :wink: